From fc846b935a7058f5b1d97407e4167e12f3fd2019 Mon Sep 17 00:00:00 2001
From: yangzh <yangzh0324@thundersoft.com>
Date: Wed, 16 Jul 2014 11:09:02 +0800
Subject: [PATCH 1/3] ARM:dts:msm: support ctp solomon ssd6030qn5

Add CTP IHV-BYD@BYD-TP1089A-V2@SSD6030QN5 device tree

Signed-off-by: yangzh <yagnzh0324@thundersoft.com>

Change-Id: I1245c84dbdfe054cb6e4732c744c956e5aad9312
---
 arch/arm/boot/dts/qcom/msm8939-qrd-skuk.dtsi |   13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)
 mode change 100755 => 100644 arch/arm/boot/dts/qcom/msm8939-qrd-skuk.dtsi

diff --git a/arch/arm/boot/dts/qcom/msm8939-qrd-skuk.dtsi b/arch/arm/boot/dts/qcom/msm8939-qrd-skuk.dtsi
old mode 100755
new mode 100644
index e9902b8..9c9d491
--- a/arch/arm/boot/dts/qcom/msm8939-qrd-skuk.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8939-qrd-skuk.dtsi
@@ -189,11 +189,22 @@
 			focaltech,fw-auto-cal;
 			focaltech,ignore-id-check;
 		};
+		solomon@48 {
+			compatible = "solomon,SSD60X0";
+			reg = <0x48>;
+			interrupt-parent = <&msm_gpio>;
+			interrupts = <13 0x2>;
+			vdd-supply = <&pm8916_l17>;
+			vcc_i2c-supply = <&pm8916_l6>;
+			solomon,reset-gpio = <&msm_gpio 12 0x00>;
+			solomon,irq-gpio = <&msm_gpio 13 0x00>;
+			solomon,fw-vkey-support;
+		};
 	};
 
 	gen-vkeys {
 		compatible = "qcom,gen-vkeys";
-		label = "ft5x06_ts";
+		label = "SSD60X0";
 		qcom,disp-maxx = <1080>;
 		qcom,disp-maxy = <1920>;
 		qcom,panel-maxx = <1080>;
-- 
1.7.9.5

